libs/langchain-google-common/src/types.ts

```diff
@@ -45,6 +45,7 @@ export interface GoogleConnectionParams<AuthOptions>
 export interface GoogleAISafetySetting {
   category: string;
   threshold: string;
+  method: string
```

Comment:
Needs to be an optional arg, otherwise anyone populating this field who upgrades will be forced to populate this too.
```suggestion
  method?: string
```